charts/newrelic-infrastructure/templates/ksm/deployment.yaml

```diff
@@ -33,6 +33,11 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with include "common.dnsConfig" . }}
+      dnsConfig:
```

Comment:
anything to add into the values for this?

Reply:
I would like to add it to the documentation of the common library instead of having to repeat everything in all the charts. WDYT?

Reply:
I fear that users will not check that, I would prefer to have them in the values/readme of the chart itself

Reply:
Solved in [#881654c](https://github.com/newrelic/nri-kubernetes/pull/383/commits/881654c7b9e3b6a104b4a9a29652cf7526e4c4f8)